lib/rubocop/cop/style/explicit_block_argument.rb

```diff
@@ -6,6 +6,9 @@ module Style
       # This cop enforces the use of explicit block argument to avoid writing
       # block literal that just passes its arguments to another block.
       #
+      # NOTE: This cop only registers an offense if the block args match the
```

Comment:
Does this change make the cop safe?

Reply:
I think so! There might be a case I'm not thinking about though.

Reply:
Well, I guess you can change `Safe` to `true` and we'll see if we're lucky. :-)

Reply:
Updated!